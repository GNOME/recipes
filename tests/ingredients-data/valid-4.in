1	g	salt	
2	kg	salt	
3	l	wine	
4	liter	milk	
5	liter	beer	
1	pound	ground beef	
2	pound	meat	
1/2	box	sticks	
