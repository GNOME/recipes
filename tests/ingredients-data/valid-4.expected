AMOUNT 1.000000
UNIT g
NAME Salt

AMOUNT 2.000000
UNIT kg
NAME Salt

AMOUNT 3.000000
UNIT l
NAME Wine

AMOUNT 4.000000
UNIT l
NAME Milk

AMOUNT 5.000000
UNIT l
NAME beer

AMOUNT 1.000000
UNIT lb
NAME Ground beef

AMOUNT 2.000000
UNIT lb
NAME Meat

AMOUNT 0.500000
UNIT box
NAME sticks

